PyRoute/Calculation/TradeCalculation.py

```diff
@@ -154,14 +168,13 @@ def calculate_routes(self):
 
         # Pick landmarks - biggest WTN system in each graph component.  It worked out simpler to do this for _all_
         # components, even those with only one star.
-        landmarks = self.get_landmarks()
-        stars = [item for item in self.galaxy.stars]
-        num_stars = len(stars)
+        landmarks = self.get_landmarks(index=True)
+        stars = [self.galaxy.star_mapping[item] for item in self.galaxy.stars]
```

Comment:
```suggestion
        landmark_star = max(self.galaxy.star_mapping.values(), key=lambda item:item.wtn)
        landmark_star.is_landmark = True       
```
Instead of creating the whole list, then sorting it, only to select the first item. Does the work for you with out the extra list, and may be faster too.